content/docs/09-registries-and-packs/6-helm-charts.md

```diff
@@ -1,89 +1,82 @@
-
-# Setting up Helm Registry with Tenant Scope
-
-Palette also provides an exclusive Helm Registry for tenants to add their private Helm charts. To create your private repo from the Palette console follow these steps:
-
-
-1. Login as a Tenant administrator, navigate to **Admin > Settings > Registries**.
 
+# Palette Helm Registry
+Helm Charts are a collection of Kubernetes resource files capable of deploying services of varying complexity. Palette provides some stable default Helm charts in its public Helm Chart Registry. 
 
-2. Click on **Add New Helm Registry** and provide the registry name, endpoint, and user credentials.
+Palette also leverages developers' custom Helm registries. You can add your own public or private Helm registries to Palette's Helm registry.
 
+The Helm Chart registry synchronizes charts with the Palette console so you can use them when you create cluster profiles.
 
-3. **Protected Mode**: Enable **Protected** mode if the user's Helm registry is deployed within a private network. Palette's orchestrator running in the tenant clusters deployed in private networks downloads and deploys charts from these protected chart registries.
-   * The user credential validation is disabled in protected mode.
-   * In protected mode, the charts are not synchronized into the management console.
-   * The users can manually input the Helm chart names and versions during Cluster Profile creation in protected mode. 
+## Prerequisite
+For security, Palette requires Helm OCI version 3.7.0 and higher.
 
+## Add a Helm Chart Registry to Palette
 
-4. **Unprotected Mode**: Disable **Protected** mode if the user's Helm registry is deployed within a public network. 
-   * The charts need to be synchronized into the management console in unprotected mode. 
-   * Charts are selected from the dropdown menu during the cluster profile creation.
+To add your private Helm Chart Registry to Palette:
 
+1. Log in as a Tenant administrator.
 
-5. Enable "**No authentication**" if the tenant choice is not to set up credentials for the registry.
-   * If the user wants to create the registry without authentication, only a custom registry name and endpoint are required for the validation to complete. 
+2. From the **Main Menu** navigate to **Tenant Settings > Registries**.
 
+3. From the **Helm Registries** tab, click **Add New Helm Registry** and type the registry name and endpoint. If the registries list is long, you may need to scroll down to see the Add link.
 
-6. Disabling the **No authentication** requires a specific username and password set up along with the custom registry name and endpoint. Upon successful registration, users can build and deploy custom charts onto the custom Helm registry and use these packs while creating Cluster Profiles.
+3. Type the name of your registry and its endpoint.
 
+3. Choose ``Protected`` mode based on whether your network is private or public:
```

Comment:
@ritawatson should this be in bold instead of code blocks? It seems like a UI element but I could be wrong.